src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.ja.xlf

```diff
@@ -632,6 +632,11 @@
         <target state="new">Add parentheses</target>
         <note />
       </trans-unit>
+      <trans-unit id="Convert_LINQ_query_to_foreach">
+        <source>Convert 'query' to 'foreach'</source>
+        <target state="new">Convert 'query' to 'foreach'</target>
```

Comment:
Great.  This matches "Convert 'for' to 'foreach'", so all our different features will be consistent.